content/docs/references/cli/_index.md

```diff
@@ -30,6 +30,7 @@ This document outlines all dellctl commands, their intended use, options that ca
 | [dellctl schedule get](#dellctl-schedule-get) | Get schedules |
 | [dellctl encryption rekey](#dellctl-encryption-rekey) | Rekey an encrypted volume |
 | [dellctl encryption rekey-status](#dellctl-encryption-rekey-status) | Get status of an encryption rekey operation |
+| [dellctl images](#dellctl-images) | Prints a Table of the container images Driver needs |
```

Comment:
Hi @VamsiSiddu-7 can you please rephrase the sentence to something like this.., "List the container images used by driver."
Thanks.

Reply:
done